src/avram/upsert.cr

```diff
@@ -92,7 +113,7 @@ module Avram::Upsert
 
   # :nodoc:
   macro included
-    {% for method in ["upsert", "upsert!"] %}
+    {% for method in ["upsert", "upsert!", "bulk_upsert"] %}
```

Comment:
Can / should this functionality just live in an overload to upsert rather than a dedicated method name?

Reply:
From our DM's last night:

> [7:31 PM] grepsedawk: Also, it might be a hot take, but I really like the idea of doing method overloading over a new method here

Definitely a strong take, but we didn't change our path forward to open up discussion about this pattern.

I'm 💯 on board with an overload